idaes/apps/grid_integration/tests/util.py

```diff
@@ -0,0 +1,278 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+from collections import deque
+import pyomo.environ as pyo
+import pandas as pd
+
+
+class TestingModel:
+
+    """
+    Simple model object for testing.
```

Comment:
Does this model come from literature (is it in one of our papers)? If so, just add a quick reference here in the comment block.